src/css/fluidplayer.css

```diff
@@ -723,6 +722,62 @@ _:-ms-lang(x),
     user-select: none;
 }
 
+.fluid_video_wrapper.fluid_player_layout_default .fluid_controls_container .fluid_controls_right .fluid_control_duration  {
+    display: flex;
+    align-items: center;
+}
+
+.fluid_button_live_indicator {
+    margin-right: 10px;
+    display: inline-flex;
+    align-items: center;
+    padding: 2px 5px;
+    background-color: red;
+    color: white;
+    border-radius: 4px;
+    font-weight: bold;
+    font-size: 11px;
+    cursor: pointer;
+}
+
+.live_circle {
+    position: relative;
+    display: inline-block;
+    width: 6px;
+    height: 6px;
+    background-color: transparent;
+    border: 1px solid #ffffff;
+    border-radius: 50%;
+    margin-left: 3px;
+}
+
+.live_circle::after {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 4px;
+    height: 4px;
+    background-color: #ffffff;
+    border-radius: 50%;
+}
+
+.fluid_button_live_indicator:hover {
```

Comment:
Same here. It doesn't really make sense that we have a hover action on the button. It will feel to the user like they can click on it.